### PR TITLE
feat(19579): Add app-version to Sentry init for release tracking

### PR DESCRIPTION
**Pull request by matthew2564**
## Ticket title
One line description
[link to ticket number]()

## Checklist
- [x] Code has been tested manually
- [x] PR title includes the JIRA ticket number
- [ ] Branch is rebased against the latest develop
- [ ] Squashed commit contains the JIRA ticket number

### CODE REVIEW

File: scripts/config/application.hybrid.ts [hunk 1]
```diff
@@ -9,6 +9,7 @@ const {
   AUTH_REDIRECT_URL,
   APP_KEY_PHONE_NUMBER,
   SENTRY_DSN,
+  SENTRY_PROJ,
```
Comment (alleypa) [inline scripts/config/application.hybrid.ts:12]: @matthew2564 should this be worded as `SENTRY_VERSION`?

File: src/app/app.component.ts [hunk 2]
```diff
@@ -108,6 +106,25 @@ export class MyApp {
     }
   }
 
+  initialiseSentry = async () => {
```
Comment (alleypa) [inline src/app/app.component.ts:109]: I would suggest a consistent use of JS/TS method declaration. Lets keep it consistent with the file Function declaration

File: src/providers/global/sync.service.ts [hunk 3]
```diff
@@ -208,4 +209,19 @@ export class SyncService {
     alert.present();
     return _throw('Something bad happened; please try again later.');
   }
+
+  public setAppVersion = async (): Promise<void> => {
```
Comment (alleypa) [inline src/providers/global/sync.service.ts:213]: same here. use Function declaration

File: scripts/config/application.hybrid.ts (hunk 1 above)
Comment (alleypa) [reply scripts/config/application.hybrid.ts:12]: @matthew2564 should this be worded as SENTRY_VERSION?

File: src/app/app.component.ts (hunk 2 above)
Comment (matthew2564) [reply src/app/app.component.ts:109]: Ok will amend to:

`async initialiseSentry () {`

File: src/providers/global/sync.service.ts (hunk 3 above)
Comment (matthew2564) [reply src/providers/global/sync.service.ts:213]: Will amend to:

`async setAppVersion () {`

File: scripts/config/application.hybrid.ts (hunk 1 above)
Comment (matthew2564) [reply scripts/config/application.hybrid.ts:12]: No this is the Sentry Project name, this is used to prefix the release name as recommended by Sentry

**Pull request closed without merging**